src/middleware/node/middleware.ts

```diff
@@ -44,6 +44,19 @@ export async function middleware(
     }
   }
 
+  if (request.headers["content-type"] !== "application/json") {
```

Comment:
Maybe check with `.startsWith()`? Just in case GitHub decides to add character encoding to the header (`Content-type: application/json; charset=utf-8`)

Reply:
I hadn't thought of that, good idea!